lang/eiffelstudio/Portfile

```diff
@@ -42,11 +42,11 @@ extract.only      ${distname}${extract.suffix}
 worksrcdir        PorterPackage
 
 checksums         ${distname}${extract.suffix} \
-                    rmd160 9d4ddebf7d4b9307205294e38fc6859ee5657359 \
- 	                sha256 2016be2e94bc2f9df6c317118c1b7125f9243e646a1e763641c1dc9243e9cb40 \
+			rmd160 6b99ed29902d34a6c18bd317466740ed61179061 \
+			sha256 5dec9c91c600c94ea34170538b311b7fad970913159284b7cf9f29a1753575c5 \
```

Comment:
FYI, since this port last updated, it's become the practice to add a `size` element in the checksums line as well.

Reply:
done